datahub/company/models/company.py

```diff
@@ -419,6 +419,30 @@ def remove_from_one_list(self, modified_by):
         self.one_list_tier = None
         self.save()
 
+    def add_export_country(self, country, status, record_date, adviser):
+        """
+        Add a company export_country, if it doesn't exist.
+        If the company already exists and incoming status is different
+        check if incoming record is newer and update.
+        """
+        export_country, created = CompanyExportCountry.objects.get_or_create(
+            country=country,
+            company=self,
+            defaults={
+                'status': status,
+                'created_by': adviser,
+                'modified_by': adviser,
+                'modified_on': record_date,
+            },
+        )
+
+        if not created:
+            if export_country.status is not status and export_country.modified_on < record_date:
```

Comment:
`is not` is relying on the statuses being the same objects here – I would think `==` is more appropriate:
```suggestion
            if export_country.status == status and export_country.modified_on < record_date:
```